.github/workflows/checks-definition.yml

```diff
@@ -0,0 +1,45 @@
+on:
+  pull_request
+
+jobs:
+  check-definitions:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checkout the repository
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

Comment:
We usually pin actions to a full length commit SHA to [harden](https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions#using-third-party-actions) the [software supply chain](https://pagopa.atlassian.net/wiki/spaces/Technology/pages/734528013/Sicurezza+della+software+supply+chain).